native/packages/react-native-bpk-component-text/src/BpkText-test.common.js

```diff
@@ -95,8 +101,11 @@ const commonTests = () => {
         BpkText.propTypes
           .emphasize(
             {
+              children: 'Lorem ipsum.',
               textStyle: 'sm',
+              // $FlowFixMe
```

Comment:
This test explicitly checks for the wrong type being passed - perhaps we can just remove it?